README.md

```diff
@@ -130,17 +130,19 @@ If something goes wrong during the version update process a rollback occurs:
 - MeiliSearch is started again.
 
 Example:
-Your current version is `v0.21.0` you want to update MeiliSearch to `v0.22.0`. Thus inside your server you import and launch the script
+Your current version is `v0.23.0` you want to update MeiliSearch to `v0.24.0`. Thus inside your server you import and launch the script
 
 ```
-sh update_meilisearch_version.sh v0.22.0
+sh update_meilisearch_version.sh v0.24.0
 ```
 
-The migration fails for whatever reason. The script uses the cached `v0.21.0` binary and the cached `data.ms` of the previous version to rollback to its original state.
+The migration fails for whatever reason. The script uses the cached `v0.23.0` binary and the cached `data.ms` of the previous version to rollback to its original state.
 
 ## 💔 Version incompatibilities
 
-Incompatibility between versions happens in some specific cases. The breaking changes should be described in the CHANGELOG of the release.
+Versions that are lower than the v0.22.0 can not be migrated. 
+
+It may also happen that versions are incompatibility between each other in some specific cases. The breaking changes should be described in the CHANGELOG of the release.
```

Comment:
Sorry, I remember I already said I did not understand this sentence and you explained to me, but I don't remember, I don't understand what we mean here again 😂

Reply:
> To understand why the versions are incompatible the changes should be described in the CHANGELOG of the release in the [meilisearch release](https://github.com/meilisearch/MeiliSearch/releases).

Do you prefer this?

Reply:
Why "should be described"? Do you mean the description of the changes is available in the changlogs of the release? The "should" looks like the user has to do something